libs/common/src/constants/retry.ts

```diff
@@ -0,0 +1,13 @@
+// Durable orchestrated functions
```

Comment:
If this is only used for durable orchestrated functions, we could just keep them in there maybe? or are we going to need these elsewhere?

I'd also say if these are specific durable orchestrated functions configuration, that we follow the same pattern as other files like `deal-submission-type` for instance to provide some context to them -- Maybe something like:

```
const DURABLE_ORCHESTRATED_FUNCTIONS = {
    RETRY_OPTIONS: {
        INTERVAL_MS: 5000;
        MAX_RETRY: 5
    }
} as const
```

Then we can remove the comment at the top of the file and have it self documenting in the constant's name? I'm a little cautious of adding a word/concept so widely used such as `retry` without an explicit name as to where it's being used if it's not used everywhere (and even then I'd probably want it wrapped in `API` or `APP` or a config file to denote where it's being used, as we can configure retry on both our app and the api calls we make).

Reply:
Good suggestion, have implemented in similar pattern as `deal-submission-type`.